### PR TITLE
[WIP] fix characteres who to follow page

### LINKED ISSUE #4650: HTML Character References shown in "who to follow"
**Describe the bug**
The "Who to follow" box displays escaped HTML character references.

**To Reproduce**

1. Sign up for a new account
2. Agree to the T&Cs
3. Scroll through the list of people to follow.

**Expected behavior**
Instead of `&#39;` display the apostrophe symbol. Instead of `&quot;,` show the quote symbol.

**Screenshots**
Screen Shot 2019-10-29 at 16 37 12
Screen Shot 2019-10-29 at 16 36 38

**Desktop (please complete the following information):**

- OS: OSX
- Browser: Chrome
- Version: 78

### PR DESCRIPTION
**Pull request by maykonmenezes**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Fix characteres in who to follow page.
## Related Tickets & Documents
Closes #4650
## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [X] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [ ] no documentation needed

### CODE REVIEW

**Comment by maykonmenezes** [comment]
I've noticed that the issue was not solved yet, I'll give it a try. I am working on the specs for this fix, any suggestions for me?

File: app/views/users/index.json.jbuilder [hunk 1]
```diff
@@ -2,7 +2,7 @@ json.array! @users.each do |user|
   json.id                     user.id
   json.name                   user.name
   json.username               user.username
-  json.summary                truncate(user.summary.presence || "Active DEV author", length: 100)
+  json.summary                user.summary.mb_chars.limit(100).to_s || "Active DEV author"
```
Comment (citizen428) [inline app/views/users/index.json.jbuilder]: I'm afraid. this won't work if a user's summary is empty:

```ruby
user.summary
#=> nil
truncate(user.summary.presence || "Active DEV author", length: 100)
#=> "Active DEV author"
user.summary.mb_chars.limit(100).to_s || "Active DEV author"
#=> NoMethodError: undefined method `mb_chars' for nil:NilClass
```

This version preserves the original behavior:

```ruby
(user.summary.presence || "Active DEV author").mb_chars.limit(100).to_s
#=> "Active DEV author"
```

**Comment by citizen428** [comment]
@maykonmenezes You can use "สวัสดี" ("hello" in Thai) as a test string. Note the difference between the two methods:

```ruby
helper.truncate("สวัสดี", length: 3)
#=> "..."
s.mb_chars.limit(3).to_s
#=> "ส"
```
The second version also doesn't return a 3 character string, but the single character "ส" comprised of 3 bytes, but since our limit is quite a bit higher that's probably still ok.

Since this happens in a JSON serializer, you can use a request spec to test for this.

File: app/views/users/index.json.jbuilder (hunk 1 above)
Comment (citizen428) [inline app/views/users/index.json.jbuilder]: I'm afraid this won't work if a user's summary is empty:

```ruby
user.summary
#=> nil
truncate(user.summary.presence || "Active DEV author", length: 100)
#=> "Active DEV author"
user.summary.mb_chars.limit(100).to_s || "Active DEV author"
#=> NoMethodError: undefined method `mb_chars' for nil:NilClass
```

This version preserves the original behavior:

```suggestion
  json.summary                (user.summary.presence || "Active DEV author").mb_chars.limit(100).to_s
```

**Comment by maestromac** [comment]
I'm sorry @maykonmenezes but fixing that issue had to be done with security (XSS) in mind so I took up the responsibility. 

Superseded by https://github.com/thepracticaldev/dev.to/pull/7675

**Pull request closed without merging**